data_steward/constants/validation/participants/participant_validation_queries.py

```diff
@@ -0,0 +1,207 @@
+RDR_SEX = 'rdr_sex'
+EHR_SEX = 'ehr_sex'
+MATCH_STATUS = 'match_status'
+MATCH = 'match'
+NO_MATCH = 'no_match'
+MISSING_EHR = 'missing_ehr'
+MISSING_RDR = 'missing_rdr'
```

Comment:
nit:  These are redefining values used in `data_steward/validation/participants/validate.py`.  Could that script just import the constants defined here so we don't have to try to keep them in sync between multiple files?